### PR TITLE
Contato#6

### PR DESCRIPTION
**Pull request by franciscoaires**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,406 +1,592 @@
...
+	font-size: 40px;
+	height: 30px;
+	text-align: center;
 }
-#notseven{
-	background-image: url("../../assets/image/uneb.jpg");
-	background-size: cover;
-	width: 25%;
+
+.linha {
```
Comment (KaioPorto) [inline assets/css/style.css]: Mesma coisa que disse em outro PR. Eu não sei como vai ficar as atividades da branch de Cayo, mas essas classes em português e outras questões de padrão, vcs podem já ir consertando. Assim como o "form.contate-nos" ali na linha 484 também. E assim por diante.

File: views/contato.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <link href="//fonts.googleapis.com/css?family=Antic" rel="stylesheet" type="text/css">
+        <link href="../assets/css/style.css" type="text/css" rel="stylesheet"/>
+    </head>
+    <body>
+        <?php include("header.inc");?>
+        <section class="content">
+            <h1 class="title">Contate-nos</h1>
+            <hr class="linha" align="center" size="1.8" color=red>
+            <form class="contate-nos">
+                <input class="nome" type="text" id="id_nome" required="required" placeholder="Digite seu nome aqui">
```
Comment (KaioPorto) [inline views/contato.php]: O required não precisa ser "required=required". Alguns navegadores requerem dessa forma antiga, como o internet explorer 1.0. Mas acho desncessário a gente preocupar-se com isso, em específico. Basta somente o required solto.
Por exemplo:
<input class="algumacoisa" type="text" required>

File: views/index.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<head>
+    <title>Home</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Antic">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="../assets/css/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```
Comment (KaioPorto) [inline views/index.php]: O mesmo que já disse antes, peguem os scripts e coloquem no arquivo da footer, que geralmente é o local que se coloca os scripts da página.

File: views/contato.php [hunk 4]
```diff
@@ -10,13 +10,13 @@
...
-            <form class="contate-nos">
-                <input class="nome" type="text" id="id_nome" required="required" placeholder="Digite seu nome aqui">
-                <input class="email" type="text" id="id_email" required="required" placeholder="Digite seu email aqui">
-                <input class="assunto" type="text" id="id_assunto" required="required" placeholder="Insira o assunto aqui">
-                <textarea class="mensagem" id="id_mensagem" required="required" placeholder="Escreva aqui sua mensagem" rows=10 cols=50></textarea>
-                <input class="enviar" type="submit" id="id_enviar" placeholder="ENVIAR">
+            <hr class="line" align="center" size="1.8" color=red>
+            <form class="contact-us">
+                <input class="name" type="text" id="id_name" required placeholder="Digite seu nome aqui">
+                <input class="email" type="text" id="id_email" required placeholder="Digite seu email aqui">
```
Comment (KaioPorto) [inline views/contato.php]: Man, devia ter falado isso antes, mas enfim. Ponha o required no fim da tag, que fica melhor de visualizar. E mais organizado já que ele é um auto-atributo, digamos assim. Tipo:

<input class="class-name" type="text" placeholder="Insira..." required>

E aproveitando, como a gente falou antes, pode pôr os scripts na footer, que depois os outros farão isso também.

File: assets/css/style.css [hunk 5]
```diff
@@ -558,33 +558,33 @@ form.contate-nos input.enviar {
...
-form.contate-nos input.enviar:hover {
+form.contact-uss input.submit:hover {
 	background-color: red;
 	color: #FFFFFF;
 }
 
 @media screen and (max-width: 680px) {
     
-    form.contate-nos input.nome {
+    form.contact-us input.name {
```
Comment (KaioPorto) [inline assets/css/style.css]: Quando você põe a classe no elemento, não precisa colocá-lo também na hora de estilizar, já que ele já foi nomeado com uma classe pra isso. Aqui poderia ser simplesmente:
contact-us name {
...
}

E se não tiver outra elemento com a classe "name" no html, vc pode simplesmente usar name { ... }
Mas se tiver, coloque do jeito que falei acima, especificando que é um elemento com a classe name dentro do form.

File: assets/css/style.css [hunk 6]
```diff
@@ -558,33 +558,33 @@ form.contate-nos input.enviar {
...
     	width: 100%;
     }
     
-    form.contate-nos input.email {
+    form.contact-us input.email {
```
Comment (KaioPorto) [inline assets/css/style.css]: Você percebe que você tá usando o id dos inputs para estilizá-los mais de uma vez com a mesma propriedade? Sendo que eles são um elemento só, além de ter uma classe?

Vc poderia colocar form input { width: 100%}.
E os outros que vc não quer com 100% ou quer colocar mais propriedades, vc usa o id correspondente e utiliza.

Inclusive, não entendi o pq de cada input ter um id.

File: views/header.inc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+
+        <input type="checkbox" id="login_toggle">
+        <div class="log-cad">
+            <div id="login">
+                    <form>
+                        Login:<br>
+                        <input type="text" name="login"><br>
+                        Senha:<br>
+                        <input type="password" name="psw"><br><br>
```
Comment (KaioPorto) [inline views/header.inc]: O mesmo que já falei sobre br. O uso de dois br seguidos me sugere que queriam afastar um elemento do outro. Use o br apenas para realmente quebrar linhas que é a função dele. Por exemplo o uso acima, para quebrar a linha de Login e Senha para o input correspondente, foi correto. Apesar que seria melhor vc colocar esse Login e Senha em uma tag <label>.
Veja se realmente é preciso esses dois br's, e se for só pra afastar o elemento do outro, use o margin do CSS pra isso.

File: assets/css/style.css [hunk 8]
```diff
@@ -114,29 +114,29 @@ header {
...
     display: block;
     font-family: Antic, sans-serif;
     height: 5%;
 }
 
 #login_submit {
-    background-color: white;
+    background-color: #FFFFFF;
     border: 2px solid #ff3d33;
     font-family: Antic, sans-serif;
```
Comment (KaioPorto) [inline assets/css/style.css]: Esses elementos com font family: Antic, já não herdam o mesmo font-family de body?

File: assets/css/style.css [hunk 9]
```diff
@@ -458,6 +458,104 @@ header {
...
+    padding-top: 100px;
+	width: 50%;
+}
+
+.title {
+    border-bottom: 1px solid red;
+	font-family: antic;
+	font-size: 40px;
+	height: auto;
+    min-width: 260px;
```
Comment (KaioPorto) [inline assets/css/style.css]: Identação, procura-se.

File: assets/css/style.css [hunk 10]
```diff
@@ -458,6 +458,104 @@ header {
...
+.first {
+	min-width: 185px;
+	width: 40%; 
+}
+
+#id_topic {
+	float: left;
+	margin-right: 90%;
+	margin-top: 5px;
+    min-width: 375px;
```
Comment (KaioPorto) [inline assets/css/style.css]: Identação

File: assets/css/style.css [hunk 11]
```diff
@@ -458,6 +458,104 @@ header {
...
+	width: calc(80% + 5px);
+}
+
+#id_message {
+	height: 159.2px;
+	margin-top: 5px;
+    min-width: 375px;
+    padding: 10px;
```
Comment (KaioPorto) [inline assets/css/style.css]: E de novo, identação aqui. :)

File: models/Message.php [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+		}
+
+
+		public function insert($connect) {
+			if (!empty($name) && !empty($email) && !empty($topic) && !empty($message)){   
+    			try {
+
+					$stm = $connect­>prepare("INSERT INTO users(name, email, topic, message) VALUES (:name, :email, :topic, :message)");
+
+					$stm­>BindValue(":name", $this­>name, PDO::PARAM_STR);
```
Comment (KaioPorto) [inline models/Message.php]: no lugar de " >" não seria "->"?

File: models/scheduling.php [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+	            $this->donateDate = array_key_exists('donateDate', $attributes) ? $attributes['donateDate'] : '';
+	            $this->donateDrive = array_key_exists('donateDrive', $attributes) ? $attributes['donateDrive'] : '';
+	        }
+	    }
+
+	    public static function selectAll($pdo) {
+		    $stmt = $pdo->query("SELECT * FROM doadoresAgenda");
+		    $stmt->execute();
+		    $results = $stmt->fetchAll(PDO::FETCH_ASSOC);
+	    	return $results;
```
Comment (KaioPorto) [inline models/scheduling.php]: Identaçãozinha aqui
